InscryptionAPI/Helpers/Extensions/CardSlotExtensions.cs

```diff
@@ -1,9 +1,75 @@
-﻿using DiskCardGame;
+﻿using System.Collections;
+using DiskCardGame;
 
 namespace InscryptionAPI.Helpers.Extensions;
 
 public static class CardSlotExtensions
 {
+    /// <summary>
+    /// Check if the slot is an opponent's slot.
+    /// </summary>
+    /// <param name="cardSlot">The slot to check.</param>
+    /// <returns>true if the slot is not a player slot.</returns>
+    public static bool IsOpponentSlot(this CardSlot cardSlot)
+    {
+        return !cardSlot.IsPlayerSlot;
+    }
+    
+    /// <summary>
+    /// Check if the card in the slot has a specific card name.
+    /// This is primarily useful if you don't have a Trait, Tribe, or other distinct attribute to check the card against.
+    /// </summary>
+    /// <param name="cardSlot">The slot to check.</param>
+    /// <param name="cardName">The name of the card check.</param>
+    /// <returns>true if the slot has a card and that card has the specific cardName.</returns>
+    public static bool CardInSlotIs(this CardSlot cardSlot, string cardName)
```

Comment:
Suggest calling this "HasCard" or "Contains" (although "Contains" is usually reserved for enumerations containing items). Just something a little shorter.